docs/source/model-registry.rst

```diff
@@ -171,10 +179,71 @@ as shown below to create a new version of the model.
         run_id="d16076a3ec534311817565e6527539c0",
     )
 
+.. _using-registered-model-aliases:
+
+Deploy and Organize Models with Aliases and Tags
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Model aliases and tags help you organize and manage models in the MLflow Model Registry.
+
+Model aliases allow you to assign a mutable, named reference to a particular version of a registered model. You can use aliases to indicate the deployment status of a model version. For example, you could assign a ``champion`` alias to the model version currently in production and target this alias in workloads that use the production model. You can then update the production model by reassigning the ``champion`` alias to a different model version.
```

Comment:
This seems to duplicate the content in https://output.circle-artifacts.com/output/job/2cbb799e-6727-4ceb-8b27-adfb0ac65270/artifacts/0/docs/build/html/model-registry.html#concepts, maybe we could link to it instead?

Reply:
Actually I think it wouldn't hurt to have duplicate content. Many customers will be directly linked to this page from our UI or other pages, in which case it would be helpful to have full context on aliases and tags here.

Reply:
I've enhance the model alias concept in the concepts section. I also added tags to the concept section. But I kept the alias and tags content in this section, not just because customers might directly visit this page here, but also because the content here focuses on model deployment and organization.

Reply:
> Many customers will be directly linked to this page from our UI or other pages, in which case it would be helpful to have full context on aliases and tags here.


Hm, but why not just link to it in the section above? The docs page is a bit long as is and there's room for some drift in the content between the two sections